src/pages/index.tsx

```diff
@@ -22,6 +23,7 @@ const Index: FC = () => {
   useEffect(() => {
     if ('tasks' in response) {
       tasks = response.tasks;
+      tasks.sort((a, b) => (+a.endsOn - (+b.endsOn)));
       const taskMap: any = [];
```

Comment:
While you are at it, can you add a proper type for the `taskMap`?
https://www.typescriptlang.org/docs/handbook/basic-types.html

Reply:
As we had discussed about this issue on the call, I will mark this as resolved.